config/locales/es.yml

```diff
@@ -278,7 +278,7 @@ es:
         is_hsa: es HSA
       taxes_owed:
         withdraw_amount_higher_than_owed: Ingresa una cantidad menor o igual a %{owed_amount}
-        withdrawal_date_deadline: Ingrese una fecha entre hoy y el 15 de abril de %{year} o antes
+        withdrawal_date_deadline: Por favor, ingrese una fecha entre hoy y hasta el %{day} de abril de %{year} inclusive
```

Comment:
are we flagging this translation for design folks to update?

Reply:
bump - not sure what the procedure for this is, but worth leaving a comment in jira story & tagging anu/madison